internal/http/services/owncloud/ocdav/delete.go

```diff
@@ -95,10 +96,10 @@ func (s *svc) handleDelete(ctx context.Context, w http.ResponseWriter, r *http.R
 			// return not found error so we do not leak existence of a file
 			// TODO hide permission failed for users without access in every kind of request
 			// TODO should this be done in the driver?
-			return http.StatusNotFound, errtypes.NotFound("Resource not found") // mimic the oc10 error message
+			//lint:ignore ST1005 mimic the exact oc10 error message
+			return http.StatusNotFound, fmt.Errorf("Resource not found")
 		}
-		// TODO path might be empty or relative...
-		return status, fmt.Errorf("permission denied to delete %v", ref.Path)
+		return status, fmt.Errorf("") // mimic the oc10 error messages which have an empty message in this case
```

Comment:
Same here, even though this  seems fishy? Why did oc10 have an empty message here?

Reply:
oc10 throws a lot of NotFoundException exceptions without a specific error message: https://github.com/owncloud/core/search?q=throw+new+NotFoundException